easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-1.7-GCC-4.6.3-hwloc-chkpt.eb

```diff
@@ -23,7 +23,10 @@ dependencies = [('hwloc', '1.5.1')]
 
 # enable checkpointing support
 withchkpt = True
-osdependencies = ['blcr-libs', 'blcr-devel']
+osdependencies = [
+    ('libcr-dev', 'blcr-devel'),
```

Comment:
On Debian-like distribution, `libcr-dev` depends on `libcr0`. So it comes by itself if devel is installed.

Isn't it the same on Red Hat-like distribution?